docs/reference/autoscaling/apis/get-autoscaling-capacity.asciidoc

```diff
@@ -238,6 +238,268 @@ The API returns the following result:
+        "total" : {
+          "storage" : 0,
+          "memory" : 0
+        }
+      },
+      "current_capacity" : {
+        "node" : {
+          "storage" : 0,
+          "memory" : 0
+        },
+        "total" : {
+          "storage" : 0,
+          "memory" : 0
+        }
+      },
+      "current_nodes" : [ ],
+      "deciders" : {
+        "frozen_existence" : {
+          "required_capacity" : {
+            "total" : {
+              "storage" : 0,
+              "memory" : 0
+            }
+          },
+          "reason_summary" : "indices []",
+          "reason_details" : {
+            "indices" : [ ]
+          }
+        },
+        "frozen_shards" : {
+          "required_capacity" : {
+            "total" : {
+              "memory" : 0
+            }
+          },
+          "reason_summary" : "shard count [0]",
+          "reason_details" : {
+            "shards" : 0
+          }
+        },
+        "frozen_storage" : {
+          "required_capacity" : {
+            "total" : {
+              "storage" : 0
+            }
+          },
+          "reason_summary" : "total data set size [0]",
+          "reason_details" : {
+            "total_data_set_size" : 0
+          }
+        }
+      }
+    },
+    "ml" : {
+      "required_capacity" : {
+        "node" : {
+          "storage" : 0,
+          "memory" : 0
+        },
+        "total" : {
+          "storage" : 0,
+          "memory" : 0
+        }
+      },
+      "current_capacity" : {
+        "node" : {
+          "storage" : 0,
+          "memory" : 0
+        },
+        "total" : {
+          "storage" : 0,
+          "memory" : 0
+        }
+      },
+      "current_nodes" : [ ],
+      "deciders" : {
+        "ml" : {
+          "required_capacity" : {
+            "node" : {
+              "storage" : 0,
+              "memory" : 0
+            },
+            "total" : {
+              "storage" : 0,
+              "memory" : 0
+            }
+          },
+          "reason_summary" : "Requesting scale down as tier and/or node size could be smaller",
+          "reason_details" : {
+            "waiting_analytics_jobs" : [ ],
+            "waiting_anomaly_jobs" : [ ],
+            "waiting_models" : [ ],
+            "configuration" : { },
+            "perceived_current_capacity" : {
+              "node" : {
+                "memory" : 0
+              },
+              "total" : {
+                "memory" : 0
+              }
+            },
+            "required_capacity" : {
+              "node" : {
+                "storage" : 0,
+                "memory" : 0
+              },
+              "total" : {
+                "storage" : 0,
+                "memory" : 0
+              }
+            },
+            "reason" : "Requesting scale down as tier and/or node size could be smaller"
+          }
+        }
+      }
+    }
+  }
 }
+----
+<1> One possible reason showing "current capacity not available".  It could 
```

Comment:
Is this meant to be a footnote? I think there's a different [way](https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/) to add a footnote.